### PR TITLE
proposal enhancement. fixing codeclimate error

### PR DESCRIPTION
**Pull request by lucianosousa**
removing code duplication
https://codeclimate.com/github/call4paperz/call4paperz/compare/proposal-enhancement

### CODE REVIEW

**Comment by lucasrenan** [comment]
LGTM 👍

**Comment by lucianosousa** [comment]
@tauil ?

File: app/controllers/proposals_controller.rb [hunk 1]
```diff
@@ -62,35 +62,23 @@ def destroy
...
+    vote_action(hash)
   end
 
   def dislike
-    if Vote.dislike(proposal, current_user)
-      notice = 'You disliked the proposal.'
-    else
-      notice = t('proposals.dislikes.fail')
-    end
+    hash = {
```
Comment (tauil) [inline app/controllers/proposals_controller.rb]: I'd say that `hash` is a terrible name. What if you do like this:
```
message = {
  like: { success: t(...), failure: t(...) },
  dislike: { success: t(...), failure: t(...) }
}
```

**Comment by tauil** [comment]
:+1:

**Pull request merged**